pkg/tsdb/v3iotsdb_integration_test.go

```diff
@@ -658,18 +661,31 @@ func TestDeleteTable(t *testing.T) {
 		expected     []tsdbtest.DataPoint
 		ignoreReason string
 	}{
-		{desc: "Should delete all table",
+		{desc: "Should delete all table by time",
 			deleteFrom:   0,
 			deleteTo:     9999999999999,
+			deleteAll:    false,
 			ignoreErrors: true,
 			data: []tsdbtest.DataPoint{{Time: t1, Value: 222.2},
 				{Time: t2, Value: 333.3},
 				{Time: t3, Value: 444.4}},
 			expected: []tsdbtest.DataPoint{},
 		},
+		{desc: "Should delete all table by deleteAll",
+			deleteFrom:   0,
+			deleteTo:     0,
+			deleteAll:    true,
+			ignoreErrors: true,
```

Comment:
Why all tests pass `ignoreErrors: true`?
IMHO, no errors should be expected during these tests